src/backend/InvenTree/order/templates/order/order_base.html

```diff
@@ -53,7 +55,7 @@
         <li><a class='dropdown-item' href='#' id='export-order'><span class='fas fa-file-download'></span> {% trans "Export order to file" %}</a></li>
     </ul>
 </div>
-{% if roles.purchase_order.change %}
+{% if roles.purchase_order.add %}
```

Comment:
Why is this role-check changed?

Reply:
This is directly related to the issue I raised regarding roles. This is where I discovered the discrepancy. A change role doesn't allow you to actually do state transitions. So I changed the role to reflect what you actually need to perform the action.

Reply:
@SchrodingersGat do we want to change the perm or this assignment?

Reply:
Just to make sure we're on the same page here:
With only the change role, you get 403 errors when pushing the buttons. So, essentially, without the add role you can't actually perform the action you're allowed to see
 State transitions are POST requests, and change roles are only permitted to use PUT/PATCH.

Reply:
![firefox_aJ6GW5JBmV](https://github.com/inventree/InvenTree/assets/7506688/63f1efc6-774c-4487-8065-27c2e96df47d)

Reply:
@matmair @SchrodingersGat This is what a user without Add gets

Reply:
I would suggest that we look into custom permissions on the API endpoints, rather than requiring `add` permission here. 

Logically this is a "change" action, right? However the natural link for a "change" action is a `PUT` or `PATCH` method. But, the API views can have custom role requirements which override this default association.

Reply:
Logically, it's a change action, yeah. As the functionality stands, it would tectnically be considered a bug when only possessing a Change role.

I can revert to "change" in the CUI template if altering API endpoints is more fruitful than changing this line. As illustrated above, trying to do state transitions while having only "change"-permissions in CUI will only lead to permission denied atm. The example is of one of the new states, but it goes for all states.

MY view was that then, using "add" more accurately reflects that you actually have the permissions required to click these buttons.